### PR TITLE
Fix Add single quotes to files in application after rebasing

### PR DESCRIPTION
**Pull request by felixkiryowa**
#### What does this PR do?

-  Removing single quotations

#### Description of Task to be completed?
- On rebasing, prettier had replaced single quotations with double quotations.

#### What are the relevant pivotal tracker stories?
[#165820365](https://www.pivotaltracker.com/story/show/165820365)

[Finishes #165820365]

### CODE REVIEW

File: src/actions/signupSigninActions.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-unused-vars */
```
Comment (adesege) [inline src/actions/signupSigninActions.js]: Remove this line and fix the linting issues that occurs

File: src/assets/icons/index.js [hunk 2]
```diff
@@ -82,7 +82,14 @@ export const Google = () => (
 
 export const Twitter = () => {
   return (
-    <div style={{ backgroundColor: '#03A9F4', borderRadius: 25, display: 'inline', padding: 6 }}>
+    <div
+      style={{
```
Comment (adesege) [inline src/assets/icons/index.js:86]: Use a class instead.

File: src/components/Articles/index.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        })
+      ) : (
+        <div>No Posts where Found !!!!</div>
+      )}
+    </div>
+  );
+};
+
+SampleArticles.propTypes = {
+  articles: PropTypes.array
```
Comment (adesege) [inline src/components/Articles/index.js]: Is this a required props? Please indicate

File: src/components/InputBox/index.js [hunk 4]
```diff
@@ -1,12 +1,14 @@
+/* eslint-disable react/jsx-no-duplicate-props */
+/* eslint-disable prettier/prettier */
 import React from 'react';
 
 import './Inputbox.scss';
 
-const InputBox = ({ name, placeholder, type, onChangeInput, value }) => {
+const InputBox = ({ name, placeholder, inputtype, onChangeInput, value }) => {
```
Comment (adesege) [inline src/components/InputBox/index.js]: Input type prop should be in camel casing instead

File: src/components/InputBox/index.js [hunk 5]
```diff
@@ -1,12 +1,14 @@
+/* eslint-disable react/jsx-no-duplicate-props */
+/* eslint-disable prettier/prettier */
 import React from 'react';
 
 import './Inputbox.scss';
```
Comment (adesege) [inline src/components/InputBox/index.js]: This should be in Pascal casing to match the casing of the component

File: src/components/NavBar/index.js [hunk 6]
```diff
@@ -1,47 +1,46 @@
...
-            >
-              Accounts
-            </a>
-            <div className="dropdown-menu">
-              <a className="dropdown-item" href="/register">
-                Register
-              </a>
-              <a className="dropdown-item" href="/login">
-                Login
+class Navbar extends Component {
```
Comment (adesege) [inline src/components/NavBar/index.js]: Use a functional component instead. Remember the smart and dumb components?

File: src/components/SigninForm/index.js [hunk 7]
```diff
@@ -4,9 +4,8 @@ import InputBox from '../InputBox';
 import Button from '../SignupButton';
 
 const Signin = () => {
-
   const onChangeEmail = e => {
-    console.log(e.target.value)
+    console.log(e.target.value);
```
Comment (adesege) [inline src/components/SigninForm/index.js]: No console logs

File: src/components/SigninForm/index.js [hunk 8]
```diff
@@ -15,28 +14,27 @@ const Signin = () => {
...
-          name="password"
-          placeholder="PASSWORD"
+          type='password'
+          name='password'
+          placeholder='PASSWORD'
           onChangeInput={e => onChangePassword(e)}
         />
-        <div className="forgot">
+        <div className='forgot'>
           <span onClick={() => alert('Open dialog')}>Forgot password?</span>
```
Comment (adesege) [inline src/components/SigninForm/index.js]: Why is there an alert statement here?

File: src/components/SignupForm/index.js [hunk 9]
```diff
@@ -37,44 +37,43 @@ const Signup = props => {
...
-          type="password"
+          type='password'
           value={signupData.confirm_password}
-          name="confirm-password"
-          placeholder="CONFIRM PASSWORD"
+          name='confirm-password'
+          placeholder='CONFIRM PASSWORD'
           onChangeInput={e => onChangeConfPassword(e)}
         />
         {signupData.error !== '' ? (
```
Comment (adesege) [inline src/components/SignupForm/index.js]: `signupData.error` can also be any falsy value aside from an empty string. Your check should take care of that also

**Review by adesege — CHANGES_REQUESTED** [review]
Seeing that you also worked on fixing some double quotes issues in this PR, you can change the PR title to reflect that as well

File: src/assets/icons/index.js [hunk 10]
```diff
@@ -1,5 +1,5 @@
-import React from 'react';
-import './index.scss';
+import React from "react";
+import "./index.scss";
```
Comment (ezrogha) [inline src/assets/icons/index.js]: Hey @felixkiryowa, you still have double quotes here

File: src/components/Articles/TrendingArticles.js [hunk 11]
```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import BookmarkImage from "../../assets/images/bookmark.png";
```
Comment (ezrogha) [inline src/components/Articles/TrendingArticles.js]: please remove double quotes, @felixkiryowa

File: src/components/Articles/ArticleContainer.js [hunk 12]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import SampleArticles from "./index";
+import TrendingArticles from "./TrendingArticles";
```
Comment (ezrogha) [inline src/components/Articles/ArticleContainer.js]: please remove double quotes, @felixkiryowa

File: src/components/Articles/index.js [hunk 13]
```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import moment from "moment";
+import PropTypes from "prop-types";
+import ArticleImage from "../../assets/images/article_image.jpeg";
```
Comment (ezrogha) [inline src/components/Articles/index.js]: please remove double quotes, @felixkiryowa

File: src/index.js [hunk 14]
```diff
@@ -1,8 +1,8 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import { Provider } from 'react-redux';
-import store from './store';
-import App from './pages/Routes';
+import React from "react";
+import ReactDOM from "react-dom";
+import { Provider } from "react-redux";
+import store from "./store";
+import App from "./pages/Routes";
```
Comment (ezrogha) [inline src/index.js]: please remove double quotes, @felixkiryowa

File: src/pages/Landing/index.js [hunk 15]
```diff
@@ -1,29 +1,39 @@
...
 
-
-const Landing = () => (
-    <div>
-      <div className="page-banner-image">
-         <img className='LandingImage' src={LandingImage} alt="Landing image" />
-      </div>
-      <div className="small-screen">
-        <div>
+class Landing extends Component {
```
Comment (ezrogha) [inline src/pages/Landing/index.js]: Hey @felixkiryowa, this should be a stateless component

File: src/assets/icons/index.js (hunk 2 above)
Comment (felixkiryowa) [reply src/assets/icons/index.js:86]: Why should I ?,  
 it's a  dumb component and we are simply exporting SVG

**Review by ezrogha — CHANGES_REQUESTED** [review]
Hey @felixkiryowa,
Kindly work on this feedback

File: src/components/SigninForm/index.js (hunk 8 above)
Comment (felixkiryowa) [reply src/components/SigninForm/index.js]: It's not yet implemented, We were just testing for the click event

File: src/actions/signupSigninActions.js (hunk 1 above)
Comment (felixkiryowa) [reply src/actions/signupSigninActions.js]: I have fixed this

File: src/assets/icons/index.js (hunk 10 above)
Comment (felixkiryowa) [reply src/assets/icons/index.js]: I have fixed this

File: src/components/Articles/index.js (hunk 3 above)
Comment (felixkiryowa) [reply src/components/Articles/index.js]: I have fixed this

File: src/components/InputBox/index.js (hunk 5 above)
Comment (felixkiryowa) [reply src/components/InputBox/index.js]: I have fixed this

File: src/components/InputBox/index.js (hunk 4 above)
Comment (felixkiryowa) [reply src/components/InputBox/index.js]: I have fixed this

File: src/components/NavBar/index.js (hunk 6 above)
Comment (felixkiryowa) [reply src/components/NavBar/index.js]: I have fixed this

File: src/components/SigninForm/index.js (hunk 7 above)
Comment (felixkiryowa) [reply src/components/SigninForm/index.js]: I have fixed this

File: src/index.js (hunk 14 above)
Comment (felixkiryowa) [reply src/index.js]: I have fixed this

File: src/pages/Landing/index.js (hunk 15 above)
Comment (felixkiryowa) [reply src/pages/Landing/index.js]: This component should connect to redux, therefore it should be a smart component

**Review by ezrogha — APPROVED** [review]
Well done with the feedback @felixkiryowa

**Comment by adesege** [comment]
@felixkiryowa Can you get at least 2 LGTMs on this PR and also fix the merge conflict?

File: src/actions/articlesActions.js [hunk 16]
```diff
@@ -0,0 +1,14 @@
...
+export const fetchArticles = () => (dispatch) => {
+	return axios
+		.get('https://ah-backend.herokuapp.com/api/articles/')
+		.then((response) => {
+			
+			dispatch({
+				type: GET_ARTICLES,
+				payload: response.data.results
+			});
+		})
```
Comment (ezrogha) [inline src/actions/articlesActions.js]: Hey @felixkiryowa, please work on this indentation should be two

File: src/components/Articles/ArticleContainer.js [hunk 17]
```diff
@@ -0,0 +1,14 @@
...
+import TrendingArticles from './TrendingArticles';
+
+const ArticleContainer = (props) => {
+	return (
+		<div className='wrapper'>
+			<SampleArticles articles={props.articles} />
+			<TrendingArticles articles={props.articles} />
+		</div>
+	);
+};
```
Comment (ezrogha) [inline src/components/Articles/ArticleContainer.js:12]: Please change your indentation @felixkiryowa

**Review by ezrogha — CHANGES_REQUESTED** [review]
Hey @felixkiryowa, Kindly work on the indentations

**Comment by adesege** [comment]
@felixkiryowa I will close this PR now as it has a lot of file changes and I am not confident about the quality of the PR because a lot of the file changes are not within the scope of the PR. Create a separate PR that addresses a specific task in this PR instead.

**Pull request closed without merging**